week-5/A-fetch/index.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
```

Comment:
@NateWr  suggested (in a different context) that we let them use a package like [serve](https://www.npmjs.com/package/serve) to serve the html file through a local server. I am just putting this here but it's up to you - a pro for that would be that they'd be introduced to npm and the idea of a server earlier, but a big drawback right now would be that they will confuse what's a server and what's a client even more)

Reply:
Hmm - as we're gonna have a lot of new concepts this lesson, can I suggest we leave `npm` for now? We can introduce at the `unit testing` lessons?

And yes, maybe it would be better if we keep the server at some external URL, and avoid introducing the concept of "local server" yet?